src/components/CheckoutModal/index.js

```diff
@@ -0,0 +1,285 @@
+    const createData = (name, amount) => {
+        return {name, amount};
+    };
+
+    const rows = burgerIngredients.map((ingredient) =>
+    createData(ingredient[0], ingredient[1]));
+
+    const handleOrder = async ( data )  => {
+        const {
+            name: orderName,
+            phone: orderPhone,
+            email: orderEmail,
+            address: orderAddress,
+        } = data;
+
+        const orderData = {
+            orderName,
+            orderPhone,
+            orderEmail,
+            orderAddress,
+            orderedPrice,
+            orderProduct: Object.fromEntries(burgerIngredients),
+            orderFastDelive: checked,
+        };
+
+        try {
+            setLoading(true);
+            await axios.post("https://burger-api-xcwp.onrender.com/orders",orderData);
+            setLoading(false);
+            setOrderStatus(true);
+            clearBurger();
+        } catch (error) {
+            setLoading(false);
+            console.error(error);
+            setOrderStatus(false);
+            clearBurger();
+        }
+    };
+    return (
+        <Dialog
+        open={isOpen}
+        onClose={handleClose}
+        fullWidth={true}
+        maxWidth="md"
+      >
+        <DialogTitleStyled>Order</DialogTitleStyled>
+        <DialogContent>
+          <br />
+          <br />
+          {orderStatus ? (
+            <OrderSuccessStyled>Order Success</OrderSuccessStyled>
+          ) : (
+            <>
+              <TableContainer component={Paper}>
+                <Table>
+                  <TableBody>
+                    {rows.map((row) => (
+                      <TableRow key={row.name}>
+                        <TableCell align="left">
+                          <Image name={row.name} />
+                        </TableCell>
+                        <TableCell
+                          style={{
+                            fontFamily: "Monsterrat Bold",
+                            fontSize: "22px",
+                          }}
+                          align="right"
+                        >
+                          {row.amount}
+                        </TableCell>
+                      </TableRow>
+                    ))}
+                  </TableBody>
+                </Table>
+              </TableContainer>
+              <br />
+              <Formik
+                initialValues={{
+                  name: "",
+                  email: "",
+                  phone: "+380",
+                  address: "",
+                  isFast: false,
+                }}
+                validationSchema={SignupSchema}
+                onSubmit={(values) => handleOrder(values)}
+              >
+                {({ isSubmitting }) => (
+                  <FormStyled>
+                    <LabelStyled>
+                      <FieldStyled
+                        validate={validationRules.validateName}
```

Comment:
Validation should be improved, it shows errors when I try to enter valid values, after entering invalid values 

https://user-images.githubusercontent.com/40695369/211357717-706bb90d-2299-4f47-a32c-23b5f278c623.mov